src/core/public/css_utils.ts

```diff
@@ -19,11 +19,9 @@ import bg_bottom_branded_dark from './styles/core_app/images/bg_bottom_branded_d
 // The `--kbnAppHeadersOffset` CSS variable is automatically updated by
```

Comment:
File renamed because the linter complained about the name not following snake_case format.